README.md

```diff
@@ -213,11 +213,7 @@ var pubsub = gcloud.pubsub({
   keyFilename: '/path/to/keyfile.json'
 });
 
-// Create a new topic.
-pubsub.createTopic('my-new-topic', function(err, topic) {});
-
-// Reference an existing topic.
-var topic = pubsub.topic('my-existing-topic');
+var topic = pubsub.topic('my-topic');
 
 // Publish a message to the topic.
```

Comment:
Maybe comment here saying Publish a message to the topic, automatically creating the topic if it doesn't exist yet.